spec/lib/protocol/waterline-spec.js

```diff
@@ -4,26 +4,37 @@
 'use strict';
 
 describe('Protocol.Waterline', function() {
-    var waterlineProtocol;
-    var collection = { identity: 'dummy' };
-
+    var waterlineProtocol,
+        collection = { identity: 'dummy' },
+        testSubscription,
+        testMessage,
+        messenger;
+        
     helper.before();
 
     before(function () {
         waterlineProtocol = helper.injector.get('Protocol.Waterline');
+        messenger = helper.injector.get('Services.Messenger');
+        var Subscription = helper.injector.get('Subscription');
+        testSubscription = new Subscription({},{});
+        sinon.stub(testSubscription);
+        sinon.stub(messenger);
     });
 
     helper.after();
 
     it('should publish a created event', function() {
+        messenger.publish.resolves(Promise.resolve());
```

Comment:
Functionally it's the same since a promise rejection will fail the test, but this way the code makes it a little clearer what the intention of the test is.